lib/savings_and_checking.rb

```diff
@@ -0,0 +1,126 @@
+module Bank
+  class SavingsAccount < Account
+    def initialize(id, balance, open_date)
+      super(id, balance, open_date)
+        if @balance < 1000
+          raise ArgumentError, "Accounts cannot be opened with less than $10. Please try again."
+        end
+    end
+
+    def withdraw(withdraw_amount)
+      transaction_fee = 200
+      temp_balance = @balance - transaction_fee.to_i - withdraw_amount.to_i
```

Comment:
I like the way you've used a local variable here to keep track of balance before you've decided to set it later